common/dependabot-common.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rubocop", "~> 1.9.0"
   spec.add_development_dependency "simplecov", "~> 0.21.0"
   spec.add_development_dependency "simplecov-console", "~> 0.9.1"
+  spec.add_development_dependency "stackprof", "~> 0.2.16"
```

Comment:
Looks like this is tripping up the python linter, might need to exclude `.bundle` when running `pyenv exec flake8`?

Reply:
Ah, we run flake8 on all folders. Should be fixed 🤞